gulp-test-chrome-debug.js

```diff
@@ -0,0 +1,80 @@
+const { series } = require('gulp');
+const del = require('del');
+const run = require('gulp-run-command').default;
+const fs = require('fs');
+
+/**
+ * Run a command in the vscode-chrome-debug working directory
+ */
+function runInDebug(cmd) {
+    return run(cmd, { cwd: './vscode-chrome-debug' })();
+}
+
+/**
+ * Delete the vscode-chrome-debug working directory (if it exists)
+ */
+function clean(done) {
+    // using sync because async del doesn't report failures correctly, can can cause hangs
+    del.sync(['vscode-chrome-debug']);
+    done();
+}
+
+/**
+ * Clone the vscode-chrome-debug project from GitHub
+ */
+function clone() {
+    return run('git clone -b v2 --single-branch --depth 1 https://github.com/Microsoft/vscode-chrome-debug.git')();
+}
+
+/**
+ * Run `npm install` on vscode-chrome-debug
+ */
+function install() {
+    return runInDebug('npm install');
+}
+
+/**
+ * Substitute our version of -core for the tests
+ */
+function setCoreVersion(cb) {
```

Comment:
You could use `npm link` in the -core folder, then `npm link vscode-chrome-debug-core` in the `vscode-chrome-debug` folder

Reply:
True, but that's just using symlinks to do the same thing, no?

Reply:
Yes, it's probably shorter and more of a standard way to do it.

Reply:
Fair. I'll change it.